src/components/pages/events/eventOverview/eventOverview.tsx

```diff
@@ -46,6 +48,11 @@ const EventOverview: React.FC = () => {
   const [pastEvents, setPastEvents] = useState<Event[] | undefined>();
   const [pastErrorMsg, setPastErrorMsg] = useState<string>('');
   const { addToast } = useToast();
+  const [skip, setSkip] = useState<number>(0);
+  const [limit] = useState<number>(10);
```

Comment:
Since this value is set to `10` and is never changed, it's better to set it to a constant value rather than a useState.